src/modules/core/components/Comment/Input/CommentInput.css

```diff
@@ -18,7 +18,7 @@
   box-shadow: shadow;
 }
 
-.commentBox textarea[aria-invalid="true"] {
+.commentBox textarea[aria-invalid='true'] {
```

Comment:
You should correct back to double quotes as everywhere else in the dapp. I guess it was changed during the ordeal with your style linter.

Reply:
Have done, thanks. Have also updated my prettier config so won't happen again!